app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -794,6 +797,7 @@
     <icon drawable="@drawable/proddy" package="co.thomasmenzel.proddy" name="Proddy" />
     <icon drawable="@drawable/progressbar95" package="com.spookyhousestudios.progressbar95" name="Progressbar95" />
     <icon drawable="@drawable/proton_calendar" package="me.proton.android.calendar" name="Proton Calendar" />
+    <icon drawable="@drawable/proton_calendar" package="org.lineageos.etar" name="Calendar" />
```

Comment:
Calendar icons are now stored in lawnchair launcher

Reply:
didn't work with me

Reply:
https://github.com/LawnchairLauncher/lawnchair/commit/aa251677483fde500a0f34fd31e761bdb44566c7

Reply:
Could you remove the calendar link please?

Reply:
what do I replace it with? @Chefski

Reply:
done, removed